src/main/java/com/minecolonies/configuration/ConfigurationHandler.java

```diff
@@ -84,7 +84,7 @@ public static synchronized void init(File file)
      * @param comment      Comment in config file
      * @return Value in the configuration file
      */
-    private static int getClampedInt(Configuration config, String category, String key, int defaultValue, int min, int max, String comment)
+    private static int getClampedInt(final Configuration config, final String category, final String key, final int defaultValue, final int min, final int max, final String comment)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)